bubblesub/ui/audio/base.py

```diff
@@ -119,10 +119,11 @@ def _create_new_subtitle(api: Api, pts: int, by_end: bool) -> None:
         pts = api.video.current_stream.align_pts_to_near_frame(pts)
     insertion_point = get_subtitle_insertion_point(api, pts, by_end)
     if api.video.current_stream:
-        insertion_point.start = api.video.current_stream.align_pts_to_near_frame(
+        current_stream = api.video.current_stream
```

Comment:
This variable is also used above, please extract it earlier in this function and name it `current_video_stream`

Reply:
Ok, thanks